alert-engine/src/test/scala/io/phdata/pulse/alertengine/AlertEngineImplTest.scala

```diff
@@ -97,21 +75,22 @@ class AlertEngineImplTest
   }
 
   test("get active alert") {
-    val alert = AlertRule("category: ERROR", 1, Some(0), List("tony@phdata.io"))
+    val a = TestObjectGenerator.alertRule(query = "category: ERROR", retryInterval = 1, resultThreshold = Some(1), alertProfiles = List("test@phdata.io"))
```

Comment:
This should have a more descriptive name than 'a', change it back to 'alert'?

Reply:
Refactored variable names and test profiles with more descriptive names